### PR TITLE
zh-translation: faq/general/how-to-join-slack.md

### PR DESCRIPTION
**Pull request by youhonglian**
[ ] Configuration Infrastructure
[X] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Review by SataQiu — COMMENTED** [review]
/ok-to-test

File: content/zh/faq/general/how-to-join-slack.md [hunk 1]
```diff
@@ -3,6 +3,4 @@ title: 如何加入 Istio Slack 工作区？
 weight: 180
 ---
 
-我们正在制定一个加入 [Istio Slack 工作区](https://istio.slack.com)的正式流程。
-目前，你需要被一个 Istio 社区的 member 邀请才能加入工作区。
-如果你不知道谁是 Istio 社区 member，那么在[这个论坛](https://discuss.istio.io/c/contributors)提问，之后会有人给你发送加入邀请。
+如果您想与我们社区的成员进行实时互动，您可以加入 [Istio Slack 工作区](https://slack.istio.io)的正式流程。
```
Comment (kebe7jun) [inline content/zh/faq/general/how-to-join-slack.md:6]:
```suggestion
如果您想与我们社区的成员进行实时互动，您可以加入 [Istio Slack 工作区](https://slack.istio.io)。
```

**Review by kebe7jun — APPROVED** [review]
lgtm

**Pull request merged**